### ISSUE TITLE
Update usePost/usePut API

### ISSUE THREAD

**Issue by mazmassa** [massalabs/station-massa-wallet#473]
Until we don't have a specific Repository/Monorepo for usePost/Patch etc we must update the usePost here in the wallet to be the same as we have in MassaStation.

We have two options:
1 - (hard option)  update the API here to include the ability to read {payload & params} as we did in MassaStation and update ALL requests that usePost to include payload.

2 - (easily option) update the API here and in MassaStation repo to be able to recognize if we pass an object as { payload } and be able to by default take the params as a { payload } object when just the params are given. This would avoid us to change each usePost in this repo.

**Comment by Pierregvx** [comment]
I think I can try to see if option 2 is doable

**Comment by Thykof** [comment]
I don't think we need to tackle this. I don't think we should modify usePost in the wallet plugin to match the one in MassaStation. Because in the wallet plugin, it would mean: write code that is not used. To me it's useless and risky because it could bring bugs and we don't need theses changes in the wallet from a user story point of view.

**Comment by longgamma-massa** [comment]
@Thykof @mazmassa what is the conclusion for this task. Should we close as not planned?

**Comment by mazmassa** [comment]
> @Thykof @mazmassa what is the conclusion for this task. Should we close as not planned?

We'll sync as soon we can to define that. But IMHO is **NOT** out of date. :)
